query-connector/src/styles/custom-styles.scss

```diff
@@ -26,7 +34,7 @@ h1 {
 
 .main-body {
   background-color: rgba(247, 249, 250, 1);
-  flex: auto; // Sizes main-body to fill space between header and footer
+  flex: 1 1 auto;
```

Comment:
could we add this to `main-container` as well, so other pages can have the footer anchored to the bottom?